### DISCUSSION TITLE
V2.0.0 released

### DISCUSSION THREAD

**Discussion by dhruvinsh** [dhruvinsh/ws-ephemeral#4] [Announcements]
Hope this new udpate help some people.

few highlights,
- on ws it will now set only matching port
- on qbit it will set listening port automatically
- also one new added feature on qbit side where it will conifgure dht, pex and lsd to disable or enable to have proper support for private tracker. some private tracker doesn't like this to be enable.

Please go through the readme and let me know.

Unraid template is updated and soon to be avalable.

**Comment by x134h34kr** [comment]
From my limited testing, it seems to be working.

Removed  any ephemeral port configurations from WS and changed the port in qBit to a random one. 

Then I started the ws-ephemeral docker container and checked logs. No errors.

Matching port was showing in Windscribe admin panel and qBit settings have been changed accordingly. 

Will monitor during the next days.

thanks for the update

**Reply by x134h34kr** [reply]
Well, I found an issue with my setup.

This container is doing exactly what is supposed to, but unless I restart my container running WS vpn after the port change, the new port is not picked up automatically (it makes sense actually).

Not sure how I'm gonna work around this one tbh ... I'm using unRaid btw, but that shouldn't matter.

**Reply by dhruvinsh** [reply]
can you explain little bit more. 

> but unless I restart my container running WS vpn

you mean qbit container running behind WS?

>  the new port is not picked up automatically
you mean new matching that setup on the WS account doesn't get picked up qbit even though qbit's lestening port set to new port settings?

**Reply by kellyinvestor** [reply]
The openvpn container (binhex/arch-qbittorrentvpn in my case) has to be restarted after windscribe changes the port.  This is possible from within a container by mounting docker.sock, but it is a security concern.

**Comment by vtruong68** [comment]
Is the variable in the unraid template, WS_EPHEMERAL_PORT, deprecated now? Or do we still need to set this?

**Reply by dhruvinsh** [reply]
If you are using older version of ws-ephemeral then it is required, but with new version you can fill any value since it is mandatory, but that value doesn't matter anymore. Pushing changes for unraid template now.
